includes/class-wc-ajax.php

```diff
@@ -1411,7 +1329,14 @@ public static function json_search_products( $term = '', $include_variations = f
 		$products        = array();
 
 		foreach ( $product_objects as $product_object ) {
-			$products[ $product_object->get_id() ] = rawurldecode( $product_object->get_formatted_name() );
+			$formatted_name = $product_object->get_formatted_name();
+			$managing_stock = $product_object->managing_stock();
+
+			if ( $managing_stock ) {
+				$formatted_name .= ' &ndash; ' . wc_format_stock_for_display( $product_object );
```

Comment:
@mikejolley this is very useful when adding items to orders manually, so 👍 for that! 

From what I can tell, the added stock details will be displayed when searching for products in other places, e.g. in Up-Sells / Cross-Sells fields, or in custom search fields using `json_search_products`. Not sure if stock data is relevant in all cases? Might be worth finding a way to control this, either by adding a parameter in the function, or by adding a filter here.

Reply:
Yeah I agree. Not sure how yet.